server/controllers/tournament/createTournament.ts

```diff
@@ -0,0 +1,11 @@
+import { Request, Response } from 'express';
+import catchAsync from '../../utils/catchAsync';
+
+const createTournament = catchAsync(
+  (req: Request, res: Response) => {
+    // waiting for database models
+    res.status(204).end();
+  },
+);
+
+export default createTournament;
```

Comment:
xD You had to copy only routes/index.ts without tournament's files to avoid conflict, but tournament is a part of the other task and here will be one big conflict